LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2010-2017 Tim Caswell
+Copyright (c) 2010-2018 Tim Caswell
```

Comment:
Thanks, but actually this doesn't need to be updated every year in order to cover the legal angle. Let's just revert it back to 2010, so we never need these PRs again.

Reply:
@ljharb - checked in. let me know changes if any